subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -420,17 +425,18 @@ int fota_download_s0_active_get(bool *const s0_active)
 }
 
 int fota_download_start_with_image_type(const char *host, const char *file,
-	int sec_tag, uint8_t pdn_id, size_t fragment_size,
+	const int *sec_tag_list, uint8_t sec_tag_count, uint8_t pdn_id, size_t fragment_size,
 	const enum dfu_target_image_type expected_type)
 {
-	static int sec_tag_list[1];
 	uint32_t host_hash = 0;
 	uint32_t file_hash = 0;
 	int err = -1;
+	static int sec_tag_list_copy[CONFIG_FOTA_DOWNLOAD_SEC_TAG_LIST_SIZE];
 
 	struct download_client_cfg config = {
 		.pdn_id = pdn_id,
 		.frag_size_override = fragment_size,
+		.sec_tag_list = sec_tag_list_copy,
```

Comment:
you could remove this line and set it down below with `config.sec_tag_count = sec_tag_count;`,  just so its all done in one place 🤷